src/store/files/actions.ts

```diff
@@ -222,7 +222,7 @@ export const actions: ActionTree<FileState, RootState> = {
             const filename = payload.requestParams.dest
                 .substr(payload.requestParams.dest.lastIndexOf('/'))
                 .replace('/', '')
-            const sourceDir = payload.requestParams.dest.substr(0, payload.requestParams.dest.lastIndexOf('/'))
+            const sourceDir = payload.requestParams.dest.substr(0, payload.requestParams.source.lastIndexOf('/'))
```

Comment:
```suggestion
            const sourceDir = payload.requestParams.source.substr(0, payload.requestParams.source.lastIndexOf('/'))
```
i think you should change both here.

Reply:
Resolved!